src/Symfony/Bridge/Twig/Resources/views/Email/zurb_2/notification/body.html.twig

```diff
@@ -49,13 +49,15 @@
         <wrapper class="secondary">
             <spacer size="16"></spacer>
             {% block footer %}
+                {% if show_footer is not defined or show_footer %}
```

Comment:
One idea that just hit me: instead of `show_footer`, we could have a `footer_text` context that defaults to the string "Notification e-mail sent by Symfony". Then, our override we set this to `null`. 

It seems just as valid as having a `show_footer` option and is a bit more flexible.